.travis.yml

```diff
@@ -36,6 +36,7 @@ env:
 
 install:
   - "pip install -r requirements.txt"
+  - "pip install -r requirements/test-local.txt"
```

Comment:
I think it does, but I'll give it another test now that I've made these changes.